php-classes/Slate/CBL/Demonstrations/StudentDashboardRequestHandler.php

```diff
@@ -122,10 +122,21 @@ public static function handleCompletionsRequest()
         $completions = [];
 
         foreach ($competencies AS $Competency) {
-            $completions[] = array_merge([
-                'StudentID' => $Student->ID,
-                'CompetencyID' => $Competency->ID
-            ], $Competency->getCompletionForStudent($Student));
+            $StudentCompetency = StudentCompetency::getCurrentForStudent($Competency, $Student);
```

Comment:
@nbey let's flip the parameter order for getCurrentForStudent so that it matches the name of the class and the order of the index